tests/cmdline/commands/test_data.py

```diff
@@ -322,6 +323,11 @@ def test_bandlistshelp(self):
     def test_bandslist(self):
         self.data_listing_test(BandsData, 'FeO', self.ids)
 
+    def test_bandslist_isolated_node(self):
+        bands = BandsData()
+        bands.store()
+        self.data_listing_test(BandsData, '<<UNKNOWN>>')
```

Comment:
Would it perhaps be easier to include the "lone" BandsData node simply in  `create_structure_bands()`?

I guess then you might also not have to modify the `data_listing_test` (?)

Reply:
You are totally right! I didn't clearly understand how function `data_listing_test` works.